jdaviz/core/loaders/importers/spectrum_list/spectrum_list.py

```diff
@@ -118,28 +329,34 @@ def combine_lists_to_1d_spectrum(wl, fnu, dfnu, wave_units, flux_units):
 class SpectrumListConcatenatedImporter(SpectrumListImporter):
     @property
     def output(self):
-        spectrum_list = super().output
-        if spectrum_list is None:
-            return
+        if isinstance(self.input, Spectrum) and self.input.flux.ndim == 2:
+            self.disable_dropdown = True
+            self.user_api.spectra = '*'
 
-        wlallorig = []  # to collect wavelengths
-        fnuallorig = []  # fluxes
-        dfnuallorig = []  # and uncertanties (if present)
-
-        for spec in spectrum_list:
-            for wlind in range(len(spec.spectral_axis)):
-                wlallorig.append(spec.spectral_axis[wlind].value)
-                fnuallorig.append(spec.flux[wlind].value)
-
-                # because some spec in the list might have uncertainties and
-                # others may not, if uncert is None, set to list of NaN. later,
-                # if the concatenated list of uncertanties is all nan (meaning
-                # they were all nan to begin with, or all None), it will be set
-                # to None on the final Spectrum
-                if spec.uncertainty is not None and spec.uncertainty[wlind] is not None:
-                    dfnuallorig.append(spec.uncertainty[wlind].array)
-                else:
-                    dfnuallorig.append(np.nan)
+        spectrum_list = self.spectra.selected
+        if spectrum_list is None or not len(spectrum_list):
+            return None
```

Comment:
When will `spectrum_list` be None?  In the case of an empty list, the for-loop below would already result in an empty spectrum being returned, but I guess returning an empty list is even better.

```suggestion
            return []
```

Reply:
spectra.selected_obj returns None which errors out in the loop, but I'll go ahead and return an empty list in that conditional.